src/maps/runGraphData.ts

```diff
@@ -0,0 +1,31 @@
+import { RunGraphData, RunGraphNode } from '@prefecthq/graphs'
+import { RunGraphDataResponse, RunGraphNodeResponse } from '@/models/api/RunGraphDataResponse'
+import { MapFunction } from '@/services/Mapper'
+
+export const mapRunGraphNodeResponse: MapFunction<RunGraphNodeResponse, RunGraphNode> = function(source) {
+  return {
+    kind: source.kind,
+    id: source.id,
+    label: source.label,
+    state_type: source.state_type,
+    state_name: source.state_name,
+    start_time: this.map('string', source.start_time, 'Date'),
+    end_time: this.map('string', source.end_time, 'Date'),
+    parents: source.parents,
+    children: source.children,
```

Comment:
Why retain snake casing here and in the data response?

Reply:
talked about this in slack. Might as well, but not going to address in this PR